v3/skyloong/dt40/dt40.json

```diff
@@ -0,0 +1,184 @@
+
+{
+  "name": "Skyloong DT40",
+  "vendorId": "0x1EA7",
+  "productId": "0x6040",
+  "keycodes": ["qmk_lighting"],
+  "menus": [
+      {
+        "label": "Lighting",
+        "content": [
+          {
+            "label": "Backlight",
+            "content": [
+              {
+                "label": "Brightness",
+                "type": "range",
+                "options": [0, 255],
+                "content": ["id_qmk_rgb_matrix_brightness", 3, 1]
+              },
+              {
+                "label": "Effect",
+                "type": "dropdown",
+                "content": ["id_qmk_rgb_matrix_effect", 3, 2],
+                "options": [
+                  "0-All Off",
+                  "1-solid_color",
+                  "2-alphas_mods",
+                  "3-gradient_up_down",
+                  "4-gradient_left_right",
+                  "5-breathing",
+                  "6-band_sat",
+                  "7-band_pinwheel_sat",
+                  "8-band_pinwheel_val",
+                  "9-band_spiral_sat",
+                  "10-band_spiral_val",
+                  "11-cycle_all",
+                  "12-cycle_left_right",
+                  "13-cycle_up_down",
+                  "14-cycle_out_in",
+                  "15-cycle_out_in_dual",
+                  "16-rainbow_moving_chevron",
+                  "17-cycle_pinwheel",
+                  "18-cycle_spiral",
+                  "19-dual_beacon",
+                  "20-raindrops",
+                  "jellybean_raindrops",
+                  "22-hue_breathing",
+                  "23-hue_pendulum",
+                  "24-hue_wave",
+                  "25-pixel_rain",
+                  "26-pixel_fractal"
```

Comment:
does not match QMK config

Reply:
OK,I have perfected it. Would you like to have another look at it?